exporters/utils.go

```diff
@@ -223,3 +227,33 @@ func RemoveElements(slice []string, drop []string) []string {
 	}
 	return res
 }
+
+func PrepareTLSConfig(caCertFile string) (*tls.Config, error) {
+	config := &tls.Config{}
+	// Get the SystemCertPool, continue with an empty pool on error
+	trustedCAs, _ := x509.SystemCertPool()
```

Comment:
never shadow an error

Reply:
Doesn't have to be an actual error, it can be a container with only the go binary present and your desired CA chain.

Are you suggesting we add a log entry like so?
```go
	// Get the SystemCertPool, continue with an empty pool on error
	trustedCAs, err := x509.SystemCertPool()
	if trustedCAs == nil {
		log.Infof("Creating a new empty SystemCertPool as we failed to load it from disk: %v", err)
		trustedCAs = x509.NewCertPool()
	}
```

I don't want to treat it as `log.Error` since it's doesn't have to be in a valid use case.